### DISCUSSION TITLE
Preview mode - Docs typo?  Does not match example

### DISCUSSION THREAD

**Discussion by mconnor** [vercel/next.js#17572] [Help]
The code sample in the  [Preview feature](https://nextjs.org/docs/advanced-features/preview-mode#step-1-create-and-access-a-preview-api-route) seems to be incorrect.  

Should be
`res.writeHead(307, { Location: `/posts/${post.slug}` })`

but in the docs it says 
` res.redirect(post.slug)`

**Comment by timneutkens** [comment]
It should probably be ```res.redirect(`/posts/${post.slug}`)```. `res.redirect` is a helper Next.js provides (similar functionality as express).
